src/nginx.c

```diff
@@ -49,6 +50,7 @@ static char   nginx_curl_error[CURL_ERROR_SIZE];
 
 static const char *config_keys[] =
 {
+  "Hostname",
```

Comment:
I initially said that you should also document this in `collectd.conf.pod`, but then realized that the equivalent Apache option is not documented either, so up to you. We could also fix the Apache plugin documentation, unless it's already fixed upstream.